GeneratorInterface/GenFilters/interface/MCParticlePairFilter.h

```diff
@@ -38,6 +38,9 @@
 namespace edm {
   class HepMCProduct;
 }
+namespace HepMC {
```

Comment:
I've just tried to remove this part, but then it does not compile... I could also explicitly include the header where this class is define: HepMC/SimpleVector.h